NMRangeSlider/NMRangeSlider.m

```diff
@@ -501,6 +493,9 @@ -(void)layoutSubviews
     }
 
     self.trackBackground.frame = [self trackBackgroundRect];
+    self.trackBackground.image = self.trackBackgroundImage;
+    self.trackBackground.layer.cornerRadius = self.thickness/2;
```

Comment:
I like options. The rounded corners were put in for a need that I was needing to solve in my own project. I can definitely see a use case for rectangles as well.